### ISSUE TITLE
Option to auto-skip errors

### ISSUE THREAD

**Issue by dteviot** [dteviot/WebToEpub#160]
Request (https://addons.mozilla.org/en-GB/firefox/addon/webtoepub-for-baka-tsuki/reviews/1085751/)
> Would greatly appreciate an option to auto-skip errors, especially "images not found" and "unable to create proper xhtml", though a simple skip all errors would still work for me :)

**Comment by dteviot** [comment]
@3dycosmo  Please give me URL(s) that has problems and the error message(s) that appear.

**Comment by dteviot** [comment]
Note to self:  See: https://www.viget.com/articles/the-dialog-element/

**Comment by c-h-u-c-k** [comment]
I agree, the image parser breaks on ch371 of https://www.wuxiaworld.com/novel/emperors-domination.
Can't get past it.

**Comment by dteviot** [comment]
@c-h-u-c-k 
I've just tried emperors-domination, and had no problems with all 2671+ chapters.
I was using version 0.0.0.97 on chrome.
There were a number of images that could not be fetched, but WebToEpub did simply skipped them and recorded the issues.

You will need to provide more details for me to help you further.
1. Describe what you do to get the error.  e.g. Does it happen if you try to fetch just chapter 371?  Does it happen if you try to fetch all chapters?
2. Describe "parser breaks" in more detail.  What does WebToEpub do?  How does this differ from what you expect.
3. Is there an error message?  if so what's the error.

**Comment by c-h-u-c-k** [comment]
Error happens on page 371 consistently. I just grabbed just 370 to 372 and it triggered. Also happens on all chapter fetch. Here's the error:

 is not a valid URL. urlHasFragment@moz-extension://ab414571-db9e-4606-a3be-23df7bdb7e42/js/ImageCollector.js:364:36
initialUrlToTry@moz-extension://ab414571-db9e-4606-a3be-23df7bdb7e42/js/ImageCollector.js:358:42
fetchImage@moz-extension://ab414571-db9e-4606-a3be-23df7bdb7e42/js/ImageCollector.js:298:31
fetchImages/temp</<@moz-extension://ab414571-db9e-4606-a3be-23df7bdb7e42/js/ImageCollector.js:108:29

**Comment by dteviot** [comment]
@c-h-u-c-k  
I think I see part of the problem. The moz-extension suggests to me that you're not using the Chrome extension (with Chrome).  I think you're using Firefox, but I'm not 100% sure.  I just tried Firefox version 72.0.1, and it had no problems.
Can you tell me what browser you are using Name and version?  (And can you run the Chrome version, in Chrome as a test, to confirm that it works?)

For my notes, it looks like the Browser you're using is not correctly handling the following:
```javascript
new URL("http://psdreview.com/wp-content/uploads/2012/11/Fantastic-Tree.jpg")
```

**Comment by c-h-u-c-k** [comment]
Using Firefox for Android latest version 68.4.1. Maybe that's the issue...
If so, maybe add a specific skip images advanced option to not even try
them? As I generally don't care about pictures, just the text.
Thanks

**Comment by dteviot** [comment]
@c-h-u-c-k 
> If so, maybe add a specific skip images advanced option to not even try
them?

It's may be a while before I can do that.  Last time I looked at implementing that, it got complicated, fast, due to interactions with other options.
Until then, are you able to use the extension with Firefox on Windows to create epubs?
